### ISSUE TITLE
How to fix it 

### ISSUE THREAD

**Issue by jooler** [fabarea/vidi#175]
thanks for your work ,
typo3 is great, but I use it so hard...
can you help me to fix it?

![cfeb48ca-28ba-4115-96c6-8f80526daa29](https://user-images.githubusercontent.com/25960405/36140165-39d346f4-10db-11e8-841a-a576ffa96053.png)
![2acc36bc-bbf3-406b-a8c9-e05f46d69fda](https://user-images.githubusercontent.com/25960405/36140169-3b666884-10db-11e8-950d-6a61920d95c0.png)

**Comment by echavaillaz** [comment]
Just apply this patch, for me this is a bug in the ext :+1: 

```
diff --git a/Classes/Module/ModulePidService.php b/Classes/Module/ModulePidService.php
index ed0c3d9..d64e4ee 100644
--- a/Classes/Module/ModulePidService.php
+++ b/Classes/Module/ModulePidService.php
@@ -26,13 +26,6 @@ class ModulePidService
      */
     protected $dataType = '';

-    /**
-     * The page record of the configured pid
-     *
-     * @var array
-     */
-    protected $page = [];
-
     /**
      * A collection of speaking error messages why the pid is invalid.
      *
@@ -188,10 +181,6 @@ class ModulePidService
      */
     protected function getPage(int $configuredPid): ?array
     {
-        if ($this->page !== null) {
-            return $this->page;
-        }
-
         $query = $this->getQueryBuilder('pages');
         $query->getRestrictions()->removeAll(); // we are in BE context.
```

Thanks,
Eric

**Comment by fabarea** [comment]
Fixed here b385a57c586cae41f7ae3fd347f14c0d40374c52
